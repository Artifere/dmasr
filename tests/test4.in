MAKE NC 8
ADD NC $1,$0
MAKE NC 16
SHL NC $0,$1
SHL NC $0,$1
SHL NC $0,$1
SHR NC $0,$1
SHR NC $0,$1
SHR NC $0,$1
SHR NC $0,$1
SETSR NC $2
SHL NC $0,$1
SHR NC $0,$1

#Ici on teste le décalage droite, ainsi que la bonne gestion des drapeaux suite à un SHR