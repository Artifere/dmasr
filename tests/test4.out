OOOWWWWÏOW